django_freeradius/tests/base/test_api.py

```diff
@@ -9,13 +9,6 @@
 
 
 class BaseTestApi(object):
-    def test_disabled_user_login(self):
-        User.objects.create_user(username='barbar', password='molly', is_active=False)
-        response = self.client.post(reverse('freeradius:authorize'),
-                                    {'username': 'barbar', 'password': 'molly'})
-        self.assertEqual(response.status_code, 401)
-        self.assertEqual(response.data, {'control:Auth-Type': 'Reject'})
-
```

Comment:
it's tricky to explain.. in short it was a mistake caused by my manual editing and a conflict with your previous patch and another patch from Aarnav, we don't need to delete this test because it's important